web/ui_utils.js

```diff
@@ -851,6 +851,9 @@ function apiPageModeToSidebarView(mode) {
 function toggleCheckedBtn(button, toggle, view = null) {
   button.classList.toggle("toggled", toggle);
   button.setAttribute("aria-checked", toggle);
+  if (button.parentElement.classList.contains("toolbarButtonWithContainer")) {
+    button.setAttribute("aria-expanded", toggle);
+  }
```

Comment:
Why is this necessary, since it feels like you're now "mixing" the `toggleCheckedBtn` and `toggleExpandedBtn` helper functions?

Also, having the query the DOM this way feels somewhat unfortunately and potentially brittle as well.

Reply:
Right, maybe using toggleExpandedBtn is the right thing to do here. I didn't change that in this patch so it's a pre-exisiting bug.